docs/output.md

```diff
@@ -42,22 +47,22 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
     - [ASCAT](#ascat)
     - [CNVKit](#cnvkit)
     - [Control-FREEC](#control-freec)
-  - [MSI status](#msi-status)
+  - [Microsatellite instability (MSI)](#microsatellite-instability-msi)
     - [MSIsensorPro](#msisensorpro)
-  - [Concatenation](#concatentation)
+  - [Concatentation](#concatentation)
```

Comment:
What is this? Is this a typo and it should be Concatenation?

Reply:
Good catch, typo was in the section, and the TOC got updated by prettier/vscode